_includes/guide/bcda_v2.html

```diff
@@ -20,14 +20,14 @@ <h2>
         </tr>
         <tr>
             <td>
-                <pre><code>curl -X GET 'https://sandbox.bcda.cms.gov/api/v2/Group/all/$export' \
+                <pre><code>curl -X GET "https://sandbox.bcda.cms.gov/api/v2/Group/all/$export" \
```

Comment:
Maybe we can sneak in highlighting of `v2`?
```suggestion
                <pre><code>curl -X GET "https://sandbox.bcda.cms.gov/api/<span style="background: yellow">v2</span>/Group/all/$export" \
```

Reply:
We can add those to either one of the static site tickets for next sprint, if there needs to be accessibility checks for it.